src/lib/containers/table/SynapseTable.tsx

```diff
@@ -158,17 +169,30 @@ export default class SynapseTable extends React.Component<
       mapUserIdToHeader: {},
       isFetchingEntityHeaders: false,
       isFetchingEntityVersion: false,
+      tableElementId: uniqueId()
     }
     this.getEntityHeadersInData = this.getEntityHeadersInData.bind(this)
   }
+  
+  // instance variables
+  resizer:any
+  
+  componentWillUpdate() {
```

Comment:
it looks like this is a deprecated method, do you know what would happen if this method was removed?

Reply:
if we do need this method then it looks like we have to use [UNSAFE_componentWillUpdate](https://reactjs.org/docs/react-component.html#unsafe_componentwillupdate)

Reply:
I'll take a look.